api/src/main/java/javax/enterprise/inject/spi/Bean.java

```diff
@@ -32,7 +32,7 @@
  * @author David Allen
  * @param <T> the class of the bean instance
  */
-public interface Bean<T> extends Contextual<T>, BeanAttributes<T> {
+public interface Bean<T> extends Contextual<T>, BeanAttributes<T>, Prioritized {
```

Comment:
Currently only a small subset of beans has a priority. Even not all alternatives, interceptors and decorators declared priority (are instead enabled and ordered in beans.xml). Therefore, we should not force this on all Bean implementations but rather make it optional. Similarly, a container should provide a Prioritized Bean impl only if it creates a Bean for a class that has @Priority.